### DISCUSSION TITLE
projects/mbtmi/img-resize

### DISCUSSION THREAD

**Discussion by giscus** [kd02109/Nextjs-Blog#42] [General]
# projects/mbtmi/img-resize

휴대폰 화면에서 결과 이미지 전체가 정확하게 나타나지 않기에 이미지 확대 기능을 구현했습니다.

https://sonblog.vercel.app/projects/mbtmi/img-resize

**Comment by juhee-playground** [comment]
이렇게 문제 정의 하고 조건들 적고 하는 방법 미리 적어두고 개발하는 방법 정말 좋네요. 좋은 습관 잘 배워 갑니다. 되게 멋있네요!!!
